prusti-tests/tests/verify_overflow/fail/type-cond-specs/in-non-trusted-function.rs

```diff
@@ -0,0 +1,11 @@
+use prusti_contracts::*;
+
+trait A {}
+
+#[refine_spec(where T: A, [
+	ensures(true)
```

Comment:
tab

Reply:
Non-autoformatted files will be my undoing…